packages/main/src/plugin/util/exec.ts

```diff
@@ -64,6 +64,15 @@ export class Exec {
 
           // convert process.env to { [key: string]: string; }'
           const sudoEnv = env as { [key: string]: string };
+          /*
+           * sudo prompt verify keys and does not support keys with special characters
+           * ( or ) on Windows
+           */
+          for (const key of Object.keys(sudoEnv)) {
```

Comment:
if it's windows specific we need to remove only on Windows

Reply:
It's not Windows specific it's just that Windows env causes the error to be raised so path is to make sure error won't be raised